client/src/reducers/user.js

```diff
@@ -9,7 +9,7 @@ export default function(state = Map({
     id: -1
 }), action) {
     switch (action.type) {
-        case RECEIVE_USER:
+        case RECEIVE_USER_SUCCESS:
             const {
                 admin,
```

Comment:
The data that flows from the api is technically `admin`. We can change the field in our js objects though. I'll do that because `admin` is a bit strange.